nemo/collections/asr/modules/rnnt_abstract.py

```diff
@@ -342,3 +342,25 @@ def mask_select_states(self, states: Any, mask: torch.Tensor) -> Any:
             states filtered by mask (same type as `states`)
         """
         raise NotImplementedError()
+
+    def batch_copy_states_mask(
```

Comment:
You can use `batch_replace_states_mask` for this (for both stateless and lstm decoders) - it is in `main` branch

Reply:
👍 Thank you